mlflow/sagemaker/__init__.py

```diff
@@ -15,6 +15,11 @@
 
 DEFAULT_IMAGE_NAME = "mlflow_sage"
 
+PYFUNC_IMAGE_URL = "707343435239.dkr.ecr.us-west-2.amazonaws.com/mlflow-pyfunc-test:latest"
```

Comment:
just to clarify, so for master, we'd make it the version of the next release, right? is that 0.2.2 now? @tomasatdatabricks

Reply:
I think it would make most sense for this to point to the current version of MLflow - i.e. mlflow.version.VERSION. That way we do not have to be managing updating versions on multiple places.